modules/shortcodes/crowdsignal.php

```diff
@@ -566,4 +602,45 @@ function crowdsignal_link( $content ) {
 wp_oembed_add_provider( '#https?://.+\.survey\.fm/.*#i', 'https://api.crowdsignal.com/oembed', true );
 wp_oembed_add_provider( '#https?://poll\.fm/.*#i', 'https://api.crowdsignal.com/oembed', true );
 
+/**
+ * Filter oEmbed HTML for PollDaddy to for AMP output.
+ *
+ * @param string $cache Cache for oEmbed.
+ * @param string $url   Embed URL.
+ * @param array  $attr  Shortcode attributes.
+ * @return string Embed.
+ */
+function crowdsignal_filter_amp_embed_oembed_html( $cache, $url, $attr ) {
```

Comment:
This oEmbed filter should probably be kept in the AMP plugin since Crowdsignal is supported by core directly:

https://github.com/WordPress/wordpress-develop/blob/14cebbe9877c4ef79ea4d79e3d8e1a5581930377/src/wp-includes/class-oembed.php#L65-L67

Reply:
Agreed.

Reply:
Implemented now in https://github.com/ampproject/amp-wp/pull/1929